test_backend.py

```diff
@@ -1,23 +1,12 @@
 import pytest
 from pathlib import Path
 
-from backend import get_robot_paths, create_robots, get_start_state, Robot, State, MovementCard, RotationCard, apply_tile_effects
+from backend import create_robots, get_start_state, Robot, State, MovementCard, RotationCard, apply_tile_effects, get_robot_names
```

Comment:
Momentálně se v testech nepoužívá `pathlib` ani `get_robot_names`. Zkusíš na to napsat test nebo založíme issue?

Reply:
novou issue a brzo doplním :-D